proteus/cmeshToolsTest.pyx

```diff
@@ -0,0 +1,581 @@
+
+import cython
+cimport cython
+import numpy as np
+cimport numpy as np
+from libcpp cimport bool
+from mpi4py import MPI
+cimport mesh as cppm
+cimport flcbdfWrappersModule as flcbdWrappersModule
+from proteus import Comm
+
+cdef class CMeshLink:
+    """
+    Need to have this intermediate class for CMesh in order to make CMesh a
+    pickable class..
```

Comment:
Do you mean pickleable here? Do we need to be able to pickle CMesh objects in order to ensure that Mesh objects themselves can be pickled? Just interested in what you're doing here.